### DISCUSSION TITLE
✅ Allow to set Responsive Breakpoint to None in the (nested) Menu

### DISCUSSION THREAD

**Discussion by SolunaCG** [elementor/elementor#21796] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

The problem I´m referring to is similar to the one that has already been discussed here: 

https://github.com/elementor/elementor/issues/7742

https://github.com/orgs/elementor/discussions/21769

Gladly an experienced user provided a solution for that problem. I´m not sure if and how that would work for the new mega menu though.

I absolutely love the new Mega Menu and I´ve already played around with its functionality a bit but what I noticed is that the whole experienced is ruined once you reach the mobile breakpoint. As I´ve already told the Elementor support and as I´ve stated in previous posts our design department is always really picky when it comes to designs, especially responsive designs. A way to design the mobile menu just like we can design the Desktop and Tablet menu with the new Mega Menu would be highly appreciated!! Those "responsive settings breakpoint" settings seem to really ruin the whole user experience and the freedom for those who create the websites, whether it´s for the new "Tabs"-widget or for the new "Mega Menu". 

As you can see on these screenshots the design breaks on mobile but works great on tablet portrait (not polished yet but it works). 

![Bildschirm­foto 2023-03-28 um 16 11 35](https://user-images.githubusercontent.com/114682524/228272316-dcc097dc-2414-482e-9364-83a1960cb0fb.png)
![Bildschirm­foto 2023-03-28 um 16 21 52](https://user-images.githubusercontent.com/114682524/228272319-3c4133d0-180d-40ab-b827-00b748b46dd2.png)

### Describe the solution you'd like

Please consider removing this functionality for the mobile versions or at least make it optional so that we can be creative and design the mobile menu just like we want! 

### Describe alternatives you've considered

Unfortunately I couldn´t find any alternatives yet. Alternatives for mobile menus seem to be very rare in general for Elementor websites. I´ve already tried multiple plugins for mobile menus but they all look pretty standard and don´t allow us to really customise the design a lot... And the "old" way of working with templates that you can toggle still isn´t very nice (at least imo). I´ve used that method on other websites but it still doesn´t "feel" good which is why I´m even more excited for the new "Mega Menu"!

**Comment by webdevs-pro** [comment]
Yes! Please add option to do not use burger as you did with regular WP menu widget.

**Comment by HanknBeanz** [comment]
Yes please! I personally can't use any widgets that force breakpoints. Like with the new Tabs, I get all excited for a release, then sit on my hands until it's useable for me.

**Comment by markobakic-srb** [comment]
Yes! This is really necessary to expand the usability of that new widget. This little adjust of the widget would give many usability options.
That way you could for example use Mega Menu as a button with dropdown content.

**Reply by brent-e** [reply]
Yes, exactly. I had planned to use it this way and spent a fair bit of time setting up the content, assuming the Breakpoint: None option was available like in the WP menu widget. I was surprised to see it wasn't.

**Comment by JoeHana** [comment]
And please add the option for horizontal scrolling like in the new taxonomy filter widtet

**Comment by webdevs-pro** [comment]
Yes please!

**Comment by yordivd** [comment]
I do not understand why this feature was omitted from this new menu element...

**Comment by joehdevwp** [comment]
Please allow to set Responsive Breakpoint to None in the (nested) Menu.  Then we can use the new Menu widget in footers and other places where we dont want a hamburger on mobile. This was an option in the old Wordpress Nav Menu. Why wouldn't you include it in the new version?

**Comment by phkoon** [comment]
The menu widget replaced by the new one included this feature. I Urgently need it to offer the option for more breakpoint than only the default 2... Valve plz fix

**Comment by phkoon** [comment]
Solved in update 3.17

**Comment by nicholaszein** [comment]
Hello everyone!

We have great news! 🙌 

### 📢 We're happy to announce that this feature was released with Elementor Pro v3.17.0! 🥳 

You can now set the **`Breakpoint`** in the **`Responsive Settings`** of the **Menu** widget to **`None`** if you want.

Thank you for contributing with your requests! 🙏 

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
